divi/src/wallet.h

```diff
@@ -8,12 +8,15 @@
 #ifndef BITCOIN_WALLET_H
 #define BITCOIN_WALLET_H
 
+#ifndef BITCOIN_UTILSTRENCODINGS_H
+#include "utilstrencodings.h" //for atoi64
+#endif // BITCOIN_UTILSTRENCODINGS_H
```

Comment:
Doesn't need a second header guard. One's already in the header

Reply:
fixed it